db/pgstorage/pgstorage.go

```diff
@@ -268,17 +268,31 @@ func (p *PostgresStorage) ConsolidateBatch(ctx context.Context, batch *etherman.
 
 // GetClaim gets a specific claim from the storage.
 func (p *PostgresStorage) GetClaim(ctx context.Context, depositCounterUser uint, networkID uint, dbTx pgx.Tx) (*etherman.Claim, error) {
-	return nil, nil
+	var (
+		claim  etherman.Claim
+		amount string
+	)
+	const getClaimSQL = "SELECT index, orig_net, token_addr, amount, dest_addr, block_id, network_id, tx_hash FROM syncv2.claim WHERE index = $1 AND network_id = $2"
+	err := p.getExecQuerier(dbTx).QueryRow(ctx, getClaimSQL, depositCounterUser, networkID).Scan(&claim.Index, &claim.OriginalNetwork, &claim.Token, &amount, &claim.DestinationAddress, &claim.BlockID, &claim.NetworkID, &claim.TxHash)
+	claim.Amount, _ = new(big.Int).SetString(amount, 10) //nolint:gomnd
+	return &claim, err
 }
 
 // GetDeposit gets a specific deposit from the storage.
 func (p *PostgresStorage) GetDeposit(ctx context.Context, depositCounterUser uint, networkID uint, dbTx pgx.Tx) (*etherman.Deposit, error) {
-	return nil, nil
+	var (
+		deposit etherman.Deposit
+		amount  string
+	)
+	const getDepositSQL = "SELECT orig_net, token_addr, amount, dest_net, dest_addr, deposit_cnt, block_id, network_id, tx_hash, metadata FROM syncv2.deposit WHERE network_id = $1 AND deposit_cnt = $2"
+	err := p.getExecQuerier(dbTx).QueryRow(ctx, getDepositSQL, networkID, depositCounterUser).Scan(&deposit.OriginalNetwork, &deposit.TokenAddress, &amount, &deposit.DestinationNetwork, &deposit.DestinationAddress, &deposit.DepositCount, &deposit.BlockID, &deposit.NetworkID, &deposit.TxHash, &deposit.Metadata)
+	deposit.Amount, _ = new(big.Int).SetString(amount, 10) //nolint:gomnd
+	return &deposit, err
 }
 
 // GetLatestExitRoot gets the latest global exit root.
 func (p *PostgresStorage) GetLatestExitRoot(ctx context.Context, dbTx pgx.Tx) (*etherman.GlobalExitRoot, error) {
-	return nil, nil
+	return p.GetLatestL1SyncedExitRoot(ctx, dbTx)
```

Comment:
why don't we return the grpc ger instead of the l1 ger?

Reply:
it's not completed yet because of missing trusted globalexitroot

Reply:
ok, just take it on mind